blueprints/disruption-budgets/README.md

```diff
@@ -0,0 +1,208 @@
+# Karpenter Blueprint: NodePool Disruption Budgets
+
+## Purpose
+
+A Karpenter’s NodePool disruption budgets can be used to rate limit Karpenter’s voluntary disruptions, if no budget is set their is a default of 10% set. When calculating if a budget will block nodes from disruption, Karpenter checks if the number of nodes being deleted is greater than the number of allowed disruptions. Budgets take into consideration voluntary disruptions through expiration, drift, emptiness and consolidation. If there are multiple budgets defined in the NodePool, Karpenter will honour the most restrictive of the budgets.
+
+By applying a combination of disruptions budgets and Pod Disruptions Budgets (PDBs) you get both application and platform voluntary disruption controls, this can help you move towards continually operations to protect workload availability. You can learn more about Karpenter NodePool disruption budges and how the Kapenter disruption controllers works in the [Karpenter documentation](https://karpenter.sh/docs/concepts/disruption/#disruption-controller).
+
+## Examples
+The following provides a set of example disruption budgets:
+
+### Only interrupt a % of Karpenter managed nodes
+
+The following Disruption Budgets says, at any-point in time only disrupt 20% of the Nodes managed by the NodePool. For instance, if there were 19 nodes owned by the NodePool, 4 disruptions would be allowed, rounding up from 19 * .2 = 3.8.
```

Comment:
Ideally we should not only explain what is the functionality, but why you would apply it and what are the consequences. an idea/example:

> Applications or systems might be more affected planned disruption caused by consolidation. Some application may find consolidation is either too aggressive, while other might be more fault tolerant and would rather optimize for cost. This configuration to only interrupt a percentage of managed nodes allows you to control the level of churn in this case. This also applies for cases like `ExpireAfter`, where there is an intent to refresh nodes

Reply:
Like it, thank you!